config/travis/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# if [ "$TRAVIS_BRANCH" != "master" ] ; then
+#   echo "This commit was made against $TRAVIS_BRANCH and not master. No build for you!"
+#   exit 0
+# fi
+
+# if [ "$TRAVIS_PULL_REQUEST" != "false" ] ; then
+#   echo "This commit was made in a pull request. No build for you!"
+#   exit 0
+# fi
```

Comment:
This will be uncommented. I'm just seeing whether everything works fine in this PR.